pennylane_lightning/lightning_qubit.py

```diff
@@ -782,6 +782,32 @@ def generate_samples(self):
 
         return M.generate_samples(len(self.wires), self.shots).astype(int, copy=False)
 
+    def generate_mcmc_samples(self, kernel_name="Local", num_burnin=100):
+        """Generate Markov chain Monte Carlo(MCMC) samples
+        Args:
+            kernel_name (str): name of transition kernel.
+            num_burnin (int): number of steps that will be dropped.
+
+        Returns:
+            array[int]: array of samples in binary representation with shape ``(dev.shots, dev.num_wires)``
```

Comment:
Can we add some additional info to the dosctring on the supported kernels for `kernel_name`?